<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AgregarReglaWeb</class>
 <widget class="QWidget" name="AgregarReglaWeb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>288</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>288</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>288</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Agregar regla manualmente</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="4" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>GUARDAR REGLA</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>CANCELAR</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QTimeEdit" name="timeEdit">
       <property name="minimumSize">
        <size>
         <width>74</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>74</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="displayFormat">
        <string>hh:mm</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>24</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Sitio Web</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="comboBox">
       <property name="minimumSize">
        <size>
         <width>92</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>92</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Texto/Palabra/Código</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTimeEdit" name="timeEdit_2">
       <property name="minimumSize">
        <size>
         <width>74</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>74</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="displayFormat">
        <string>hh:mm</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Tiempo de inicio</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Tiempo de finalización</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Acción</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2" rowspan="2">
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Habilitar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Cantarell</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>REGLA DE FILTRADO</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
